packages/actor-rdf-source-identifier-file-content-type/lib/ActorRdfSourceIdentifierFileContentType.ts

```diff
@@ -28,9 +28,9 @@ export class ActorRdfSourceIdentifierFileContentType extends ActorRdfSourceIdent
     const httpAction: IActionHttp = { context: action.context,
       init: { headers, method: 'HEAD' }, input: action.sourceValue };
     const httpResponse: IActorHttpOutput = await this.mediatorHttp.mediate(httpAction);
-    if (!httpResponse.ok || !httpResponse.headers.has('Content-Type')
-      || !this.allowedMediaTypes.find((mediaType: string) => httpResponse.headers
-        .get('Content-Type').indexOf(mediaType) >= 0)) {
+    if (!httpResponse.ok || (httpResponse.headers.has('Content-Type')
+      && !this.allowedMediaTypes.find((mediaType: string) => httpResponse.headers
+        .get('Content-Type').indexOf(mediaType) >= 0))) {
```

Comment:
What about files with no content type and no valid file extension?

Reply:
Those will simply fail :-)

We'll need this for that: https://github.com/comunica/comunica/issues/289